core/templates/dev/head/pages/exploration_player/conversation_skin_embed_directive.html

```diff
@@ -664,13 +669,10 @@
       }
 
       .conversation-skin-supplemental-card-container {
-        left: 50%;
```

Comment:
Sorry for review this to late, I am on vacation.

But this will break other things, I didn't try but I am sure this will introduce an issue when viewport is narrow and an exploration has supplemental card.